proto/oracle/v1/query.proto

```diff
@@ -70,6 +71,13 @@ service Query {
       returns (QueryRequestPriceResponse) {
     option (google.api.http).post = "/oracle/request_prices";
   }
+
+  // RequestPool queries the request pool information corresponding to the given
+  // port, channel, and request key.
+  rpc RequestPool(QueryRequestPoolRequest) returns (QueryRequestPoolResponse) {
+    option (google.api.http).get =
+        "/oracle/v1/request_pool/{request_key}/{port_id}/{channel_id}";
```

Comment:
`request_key` may be considered a secret, so I'm not sure whether it is appropriate to put `request_key` in HTTP URL.

Is it possible to not include the `request_key` in the URL but still include in `QueryRequestPoolRequest`?

Reply:
What about using the sender address on the counterparty chain as the request key?